libcompiler_rt/Makefile: No such file or directory

When reinstalling the latest LiteX from master via pip, I noticed that SoCs wouldn't build anymore:
```
make: /home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/software/libcompiler_rt/Makefile: No such file or directory
```
 
Digging further, `litex/soc` from my installation did not contain a `software` folder. [_litex/soc/software_](https://github.com/enjoy-digital/litex/tree/master/litex/soc/software) is not a Python module so I suspect that the issue comes from https://github.com/enjoy-digital/litex/commit/3aee8a5227e20e37fef29936b3634360e48264ce as `graft litex/soc/software` must be kept in the MANIFEST.

I am verifying if this is enough to fix the issue and will make a PR.

Full trace:
```
INFO:SoC:--------------------------------------------------------------------------------
make: Entering directory '/home/duranda/devel/liteboot/soc_basesoc_ecp5_evn/software/libcompiler_rt'
make: /home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/software/libcompiler_rt/Makefile: No such file or directory
make: *** No rule to make target '/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/software/libcompiler_rt/Makefile'.  Stop.
make: Leaving directory '/home/duranda/devel/liteboot/soc_basesoc_ecp5_evn/software/libcompiler_rt'
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex_boards/targets/ecp5_evn.py", line 79, in <module>
    main()
  File "/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex_boards/targets/ecp5_evn.py", line 72, in main
    builder.build(run=args.build)
  File "/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/integration/builder.py", line 207, in build
    self._generate_rom_software(not self.soc.integrated_rom_initialized)
  File "/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/integration/builder.py", line 187, in _generate_rom_software
    subprocess.check_call(["make", "-C", dst_dir, "-f", makefile])
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['make', '-C', '/home/duranda/devel/liteboot/soc_basesoc_ecp5_evn/software/libcompiler_rt', '-f', '/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/software/libcompiler_rt/Makefile']' returned non-zero exit status 2.
```

There is another issue:
```
make: *** No rule to make target '/home/duranda/devel/liteboot/venv/lib/python3.8/site-packages/litex/soc/cores/cpu/serv/crt0.S', needed by 'crt0-ctr.o'.  Stop.
```

CPU verilog sources has been moved to dedicated repos. However, _cpu_ folders still have some non Python sources that must be included in the manifest.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,1 +1,11 @@
 graft litex/build/sim
+graft litex/soc/software
+graft litex/soc/cores/cpu/blackparrot
+graft litex/soc/cores/cpu/lm32
+graft litex/soc/cores/cpu/microwatt
+graft litex/soc/cores/cpu/minerva
+graft litex/soc/cores/cpu/mor1kx
+graft litex/soc/cores/cpu/picorv32
+graft litex/soc/cores/cpu/rocket
+graft litex/soc/cores/cpu/serv
+graft litex/soc/cores/cpu/vexriscv

